python/pyspark/sql/context.py

```diff
@@ -63,7 +63,33 @@ def toDF(self, schema=None, sampleRatio=None):
         """
         return sqlContext.createDataFrame(self, schema, sampleRatio)
 
+    def schema(self, datatype):
+        """
+        Converts current :class:`RDD` into a :class:`DataFrame` according to the given data type.
+
+        :param datatype:
+            a :class:`DataType` or a data type string.
+        """
+        if isinstance(datatype, StructType):
+            schema = datatype
+            def to_row(obj):
+                _verify_type(obj, datatype)
```

Comment:
Since Python and Scala/Java have different range of int/long numbers, does `_verify_type` check if the actual value is within the allowed range?